### ISSUE TITLE
Backup and restore to running remote machine

### ISSUE THREAD

**Issue by Zolmeister** [influxdata/influxdb#3937]
Having to shut-down the database to restore a backup does not work with our docker cloud setup where availability is important.
Also, it seems silly to require snapshots to be enabled in config. All production environments should be running data backups.

A more powerful remote backup solution would be extremely useful.
See RethinkDB for an example of a simple but effective backup strategy

http://www.rethinkdb.com/docs/backup/

```
rethinkdb dump -c HOST:PORT
rethinkdb restore -c HOST:PORT rethinkdb_dump_DATE_TIME.tar.gz
```

**Comment by jwilder** [comment]
Fixed via #5224

**Comment by digithunz** [comment]
There is no remote restore at the moment, will that be implement as well?
